src/entities/activity/api/useActivityAnswersMutation.ts

```diff
@@ -141,9 +138,16 @@ export const sendAnswers = async (body: SendAnswersInput) => {
   await wait(100);
 
   const data = await uploadAnswerMediaFiles(body);
+
   const encryptedData = encryptAnswers(data);
 
-  return AnswerService.sendActivityAnswers(encryptedData);
+  const result = await AnswerService.sendActivityAnswers(encryptedData);
```

Comment:
if sendActivityAnswers fails with 400+ or 500+ then this function will also fail.

Please use here promise instead, so that useMutation can handle onFailure event properly:

```
return AnswerService.sendActivityAnswers(encryptedData).then(
  result => {
    MediaFilesCleaner.cleanUpAnswerList(body.answers);

    return result;
  }
)
```